### DISCUSSION TITLE
What does `ar.Fetch()` mean in the document ?

### DISCUSSION THREAD

**Discussion by furusax0621** [lestrrat-go/jwx#881] [Q&A] [answered]
Hi,

The following document describes how to use the keyset:
https://github.com/lestrrat-go/jwx/blob/develop/v2/docs/04-jwk.md#auto-refreshing-remote-keys

> The returned `keyset` will always be "reasonably" new. It is important that
> you always call `ar.Fetch()` before using the `keyset` as this is where the refreshing occurs.

However, the variable `ar` does not appear anywhere in the sample code of this document, and there is no `Fetch()` method in `jwk.Cache`. What does `ar.Fetch()` mean ? Based on the context, is it referring to `c.Get()` ?

**Comment by lestrrat** [comment]
@furusax0621 Sorry for the late reply. That is likely a left over from the v1 version of the document. I'll check and fix.

**Comment by lestrrat** [comment, marked as answer]
@furusax0621 after re-reading, that particular comment did not need to exist, at least not in that context. The method that is most similar to `ar.Fetch()` is `c.Refresh()`, but that method is explained in an earlier section. So I have deleted that sentence all together.

Thanks for the pointer!

**Reply by furusax0621** [reply]
Thank you for your help !! 👍
